crd/fleet/fleet-crds.yaml

```diff
@@ -4072,11 +5694,37 @@ spec:
                 description: 'Internal: Whether the resource was reconciled by the
                   controller.'
                 type: boolean
+              startTime:
+                description: StartTime is the time that the failover operation started.
+                format: date-time
+                type: string
+              state:
+                description: State is the current state of the failover operation.
+                  The values are `InProgress`, `Success`, `Failed_RollbackInProgress`,
+                  `Failed_RollbackSuccess`, `Failed_RollbackFailed` `InProgress` means
```

Comment:
- Add "and" after `Failed_RollbackSuccess`
- Add a period after `Failed_RollbackFailed`